app/templates/_gulpfile.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var gulp = require('gulp');
+var gutil = require('gulp-util');
```

Comment:
You could use gulp-load-plugin, and then `$.util` instead of `gutil`, to be consistent with the rest.